packages/p2p/src/stores/order-store.js

```diff
@@ -195,17 +194,7 @@ export default class OrderStore {
         });
     }
 
-    getAdvertiserInfo(setShouldShowCancelModal) {
-        requestWS({ p2p_advertiser_info: 1 }).then(response => {
-            if (response.error) {
-                this.setErrorMessage(response.error.message);
-            } else {
-                this.setCancelsRemaining(response.p2p_advertiser_info.cancels_remaining);
-            }
-        });
-        this.getWebsiteStatus(setShouldShowCancelModal);
-    }
-
+    @action.bound
```

Comment:
`nit` Should this be omitted, since we are now using Mobx 6

Reply:
Yes, it should